docs/schema.json

```diff
@@ -0,0 +1,154 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Playdate Squad Achievements Schema",
+	"description": "The necessary structure for a game's achievements.json file.",
+    "required": [
```

Comment:
Nit: might be nice to keep the order of the required properties the same as their property definitions below. (Same applies to the achievement items.)